assets/js/steps/bibbox.js

```diff
@@ -37,6 +41,23 @@ function Bibbox({ boxConfigurationInput, machineStateInput, actionHandler }) {
         boxConfig: { get: boxConfigurationInput }
     };
 
+    /**
+     * Play birthday music if user has birthday.
+     */
+    useEffect(() => {
+        if (user === undefined) return;
+
+        const whenWasItLastPlayed = window.localStorage.getItem(user.id);
+        const lastPlayedForUser = whenWasItLastPlayed ? new Date(parseInt(whenWasItLastPlayed)) : undefined;
+        const today = new Date();
+        if (user.birthdayToday && lastPlayedForUser?.getFullYear() !== today.getFullYear()) {
+            window.localStorage.setItem(user.id, Date.now());
+            if (boxConfigurationInput.soundEnabled) {
```

Comment:
Move this if-statement out to encapsulate all the birthday code instead, to avoid evaluation if sound is not enabled.